test/test_jit.py

```diff
@@ -17232,6 +17232,60 @@ def reassign_nested():
         with self.assertRaisesRegex(RuntimeError, "previously has type"):
             self.checkScript(reassign_nested, (), optimize=False)
 
+    def test_min_single_arg_list(self):
+        def jit_min_list(a):
+            # type: (List[int]) -> int
+            return min(a)
+
+        self.checkScriptRaisesRegex(jit_min_list, ([],), Exception,
+                                    "arg is an empty sequence")
+
+    def test_min_list(self):
```

Comment:
nit: combine these `test_min_list` and `test_max_list`  and then rewrite as a pairwise iteration over a list of lists ? would be much shorter

Reply:
it's a little tricky since I have empty-list tests and those will get typed as `Tensor[]`. Let me see if I can make it look a bit better.

Reply:
I don't think they will get typed as Tensor[] since they're inputs to the function